commands/project/cloneApp.ts

```diff
@@ -79,7 +79,7 @@ exports.handler = async options => {
       });
       appId = appIdResponse.appId;
     }
-    const { name, dest } = await createProjectPrompt('', options, true);
+    const { name, dest } = await createProjectPrompt(options);
```

Comment:
This is the new syntax for `createProjectPrompt` if all we need is a name and dest from it. Alternatively, you can also pass it a list of available templates and it will prompt you to select one (see usage over in the project create command).